en/assembly/quick_start_pixhack_v3x.md

```diff
@@ -0,0 +1,61 @@
+#Pixhack V2/V3/V3X wiring quick start
```

Comment:
Consistent headings. This should be first-letter of every major word capitalised. 

> Pixhack V2/V3/V3X Wiring Quick Start